BLL/SearchDrink.cs

```diff
@@ -80,5 +80,23 @@ public static List<Drink> SearchByIngredients(SortedSet<string> ingredientsToSea
 
             return drinksFound;
         }
+        /// <summary>
+        /// Searches drinks list by specified alcohol content criteria
+        /// </summary>
+        /// <param name="alcoholicInfo"></param>
+        /// <param name="drinks"></param>
+        /// <param name="contemporaryList"></param>
+        /// <returns></returns>
+        public static List<Drink> SearchByAlcoholContent(string alcoholicInfo, List<Drink> drinks, List<Drink> contemporaryList)
```

Comment:
Why do we pass contemporaryList to this method ? it could be initialized inside the method.

Reply:
Because of 4th and 5th search option - they include both Alcoholic & optional alcohol as well Non alcoholic & optional alcohol - that's why we pass contemporaryList, because we run search two times in these cases, so we have to store list from 1st search somewhere